### ISSUE TITLE
Feature Request: filter option to show only unread messages

### ISSUE THREAD

**Issue by MarcusWolschon** [jocmp/capyreader#646]
### Steps to Reproduce

Allow to hide the (hundreds of) read messages to see the (dozens of) unread ones in the feeds.
Using the Play Store release with a FreshRSS server.

( "Filter UI" #28 was closed by an unrelated issue.)

### Expected behavior

...

**Comment by jocmp** [comment]
hey @MarcusWolschon - there is an option to filter to unread articles only. The issue https://github.com/jocmp/capyreader/issues/28 was tracking when I added it. 

<img src="https://github.com/user-attachments/assets/b690c904-c6d6-4b8a-af2a-da719b13a291" width="300px">

**Comment by MarcusWolschon** [comment]
Long-press on "all" does not open any context menu in version 2024.12.1087.
It may have become broken at some point.

**Comment by MarcusWolschon** [comment]
ARGHHHH, you click on the TITLE of the navigation drawer.
That's usually a purely informative entry with no function at all.
Given that it has the same text as the currently selected entry, it never occured to me that it was anything but a title.

**Comment by jocmp** [comment]
yep, it's the title of the drawer. Looks like you found it, but another user had suggested something similar in https://github.com/jocmp/capyreader/issues/529. I tried to explain the trade-offs with that approach in that issue as well.

An alternative approach I've explored is making this a [bottom nav/nav rail](https://m3.material.io/components/navigation-rail/overview). That would make the interaction more like Reeder Classic. It's still an option down the line, but for now I want to keep things as-is.

<img src="https://github.com/user-attachments/assets/df88ff01-e59d-4861-b5ae-aa35458c5ba5" width="300px">
